apps/nextjs-website/src/lib/strapi/solutionsListCodec.ts

```diff
@@ -61,13 +61,13 @@ const makeStrapiSolutionsListPopulate = () =>
         ],
       },
       features: {
-        populate: ['bannerLinks', 'items.icon'],
+        populate: ['items.icon'],
       },
     },
   });
 
 export const fetchSolutionsList = fetchFromStrapi(
```

Comment:
I think you should also rename this constant and fix singular/plural issue about `solution list` in all of the components and methods of the project